app/test_main.py

```diff
@@ -1,3 +1,41 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, result",
+    [
+        (14, 14, [0, 0]),
```

Comment:
I'd prefer to pass ids directly into parametrize args, like :
```suggestion
    "cat_years, dog_years, expected, test_id",
    [
        (0, 0, [0, 0], "0 cat/dog should convert into [0, 0]"),
    ]
)
```
so we can just throw it as a message in assertion after?

Reply:
yes we can and I like this option, but why do we need the id parameter?